src/components/Hero/HeroInfobox.tsx

```diff
@@ -0,0 +1,219 @@
+import {
+  type HeroSideProps,
+  type HeroButtonProps,
+  type HeroHeadingProps,
+  type HeroFloatingProps,
+} from "./Hero"
+import { HeroDropdown, type HeroDropdownProps } from "./HeroDropdown"
+
+type HeroInfoboxVariants =
+  | HeroSideProps["variant"]
+  | HeroFloatingProps["variant"]
+
+export interface HeroInfoboxProps extends HeroHeadingProps, HeroButtonProps {
+  alignment?: "left" | "right"
+  backgroundColor?: "black" | "white" | "gray"
+  size?: "sm" | "md"
```

Comment:
a bit of a controversial opinion but I feel like the `size` prop doesn't really make sense.
it feels like it should be dependent on the content as well as the screen size instead

it will also allow us to reason about our css and responsive behaviour easier as compared to the jankness below atm

<img width="1109" alt="Screenshot 2024-03-01 at 6 14 55 PM" src="https://github.com/isomerpages/isomer-components/assets/77665182/b1612bea-2784-4ec0-9298-18e0b76eecd2">